### PR TITLE
Re-activate e2e tests for the shortcode split UPE

### PR DESCRIPTION
**Pull request by timur27**
#### Changes proposed in this Pull Request
This PR removes the redundant setup logic in the shortcode split UPE test, which, in turn, fixes the test being flaky.

#### Testing instructions

* All E2E test groups should be green.

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Review by bborman22 — APPROVED** [review]
I haven't had much luck getting these tests to run locally (not because of this PR, just with my local setup) so I can't confirm it locally, but I do see all tests passing in the GH Actions. I also was following the discussion around why taking this approach and the future plans to re-enable these tests. I'll approve based on that, but if someone else on the team wanted to take a look locally if there is any value in that as I didn't want to hold this PR up any longer.

**Comment by timur27** [comment]
Thanks, @bborman22. I analyzed the tests once more and found out that the reason behind these tests being flaky is probably not SEPA. 

If we focus on the first three lines in the test
```js
await setupProductCheckout(
    config.get( 'addresses.customer.billing' )
);
await shopper.goToCheckout();
await uiUnblocked();
```

we'll find out, that `setupProductCheckout` has its own `await shopper.goToCheckout(); await uiUnblocked();` inside. This makes 
```js
await shopper.goToCheckout();
await uiUnblocked();
```
redundant, and I guess this is why the test behaves unstable occasionally. E2E tests run locally vs with Actions seem to have a bit different setup, which is why it was so hard to establish the real reason behind the problem at the first attempt.

Having all this said, I'll now remove 

```js
await shopper.goToCheckout();
await uiUnblocked();
```

entirely, and this PR shouldn't be related with SEPA. Please let me know, if all this makes sense or if there are any other questions -- I'll be happy to answer them.

**Review by bborman22 — APPROVED** [review]
I can see the redundancy you mention in those calls and could see how that would cause some flaky-ness. I had a lot more success with running my tests locally and was able to get the `upeSplit` test suite to pass with these changes now same as within GH Actions.

LGTM!

**Pull request merged**
